LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Exercism
+Copyright (c) 2022 Exercism
```

Comment:
We might need to have a range instead of a single date.  Copyright exists on original material at the time of creation, but those things that are created after the initial date, then are copyright beyond the first date.  For notice of intent of enforcement of rights, then, there may be a need to correctly indicate first publication year.  Since these things are not created and published at once, the range makes sense to me, even though I am not a lawyer.

I do not think we should be tossing out the earlier year as that might be deemed not on the up and up, really, since it is misleading. (Perhaps some of this information was created and even unmodified since 2013).

Like I said, I am not a lawyer, and surely not a lawyer that is familiar with the complexities of international copyright.  But it does not make sense that only by changing a year, we have extended the copyright of material by another 3 years of material that would have normally come under public domain because of time passing.

Reply:
Hi Kotp, Here I'm not changing terms of license  it self **License != Copyright**,  Copyright on code is renewable if we made material changes to the code and It is not strictly speaking, related to the license.

if we applied a range instead, it would be at the time of creation not the date it was published thought this isn't necessary.

Reply:
To be clear, I did not state that the terms of license were being changed, only the notice of the copyright.  I also did not indicate that licensing is copyright, it is not.  The only thing changed on this line is the copyright notice.  I am asking to be careful that notice is properly given if we are giving notice at all (It is not required for copyright to be in effect, in my jurisdiction).

I did say "first publication year" but I meant "first copyright event" which would be creation.

Reply:
Well, thank you for clearing that up. Now i can see that by doing so i you’d be creating an inaccurate copyright notice.